nativelib/src/main/resources/gc/immix/Heap.h

```diff
@@ -33,8 +33,8 @@ static inline bool heap_isObjectInHeap(Heap *heap, Object *object) {
     return Heap_IsWordInHeap(heap, (word_t *)object);
 }
 
-void Heap_Init(Heap *heap, size_t initialSmallHeapSize,
-               size_t initialLargeHeapSize);
+void Heap_Init(Heap *heap, size_t minSmallHeapSize,
+               size_t initialLargeHeapSize, size_t maxHeapSize);
```

Comment:
There is a problem with large heap size not being a part of the min/max heap size limits. It's not easy to fix though, but it's something we need to in one of the next PRs, opened an issue here: #1358

Reply:
This is actually pretty bad. Imagine someone doing -Xms1G -Xmx1G expecting a constant heap size of 1G, but it's not going to work at all because large object space will default to 1M and won't able to grow if a 2M allocation is requested.